src/components/layout.js

```diff
@@ -2,6 +2,7 @@ import PropTypes from 'prop-types';
 import React from 'react';
 
 import Header from './header';
+import Footer from './footer'
```

Comment:
I'd like it if we could include semicolons at the ends of lines in JS files.

Reply:
Sure! Good to be consistent. Will change that asap.